subsys/canbus/isotp/isotp_internal.h

```diff
@@ -27,11 +27,11 @@
  * ST      Separation time
  * PCI     Process Control Information
  */
-
+#define ISOTP_USE_CAN_FD 1
```

Comment:
defining the ISOTP_USE_CAN_FD, then ISOTP_USE_CAN_FD is defined : line 32 is true